plugins/arDominionB5Plugin/modules/browse/templates/hierarchySuccess.php

```diff
@@ -1,19 +1,20 @@
 <?php decorate_with('layout_1col.php'); ?>
 <?php slot('title'); ?>
 
-  <div class="d-flex justify-content-end flex-wrap gap-2 ms-auto">
+  <h1>
+    <?php echo __('Hierarchy'); ?>    
+  </h1>
+
+  <div class="d-flex flex-wrap gap-2 mb-3">
     <input type="button" id="fullwidth-treeview-reset-button" class="btn btn-sm atom-btn-white" value="<?php echo __('Reset'); ?>" />
-    <input type="button" id="fullwidth-treeview-more-button" class="btn btn-sm atom-btn-white" data-label="<?php echo __('%1% more'); ?>" value="" />
+    <input type="hidden" id="fullwidth-treeview-more-button" class="btn btn-sm atom-btn-white" data-label="<?php echo __('%1% more'); ?>" value="" />
```

Comment:
Why type hidden? Maybe just d-none to be updated later from the JS?

Reply:
Removed. This should not have been in here.